plugin/completion/bin_complete.py

```diff
@@ -128,7 +131,15 @@ def info(self, tooltip_request):
 
         Please use libclang or set "show_type_info" to false.
         """
-        sublime.error_message(msg)
+        # This error message can also occur when lib_complete completer cannot
+        # be initialized. This can happen for newly created files for folders
+        # that have cached cmake output. Therefore, it makes more sense to log
+        # here instead of a pop up error message.
+        if not self.use_libclang:
+            sublime.error_message(msg)
+        else:
+            log.info("clear cmake cache.")
```

Comment:
This will do nothing, right? Maybe we should just call an action to clear the cache here? Is this what you meant?